CloudFormationWindows/lansa-master-win.cfn.template

```diff
@@ -777,26 +799,47 @@
                     "26TriggerIcingUpdate"        : { "Ref" : "26TriggerIcingUpdate"},
                     "27TriggerPatchInstall"       : { "Ref" : "27TriggerPatchInstall"},
                     "28PatchBucketName"           : { "Ref" : "28PatchBucketName"},
-                    "29PatchFolderName"           : { "Ref" : "29PatchFolderName"}
+                    "29PatchFolderName"           : { "Ref" : "29PatchFolderName"},
+                    "MaxInstanceLifeTime"         : 5270400
```

Comment:
The origin of this number is unclear. It would be good to comment on it. (Same with other webserver)
I have read there is a way to put in comments using UserData. Can you do that? State the following:

For the DB ASG 
"The value 5,270,400 seconds = 61 days. This is a prime number to reduce the likelihood that this instance will be unavailable when the Web ASG instance is also unavailable"

For the Web ASG 
"The value 5,788,800 seconds = 67 days. This is a prime number to reduce the likelihood that this instance will be unavailable when the DB ASG instance is also unavailable"

61 and 67 are prime numbers close to 2 months or 62 days. Using prime numbers ensures that the DB instances and Web instances will not be refreshed on the same day as no multiple of 61 = a multiple of 67.

Reply:
Sure @robe070, I'll do it.

Reply:
Somehow your changes have been uploaded into S3 support/scalable template directory. Why is this? It needs to be a controlled location. Not used for testing. Did you do it or is the pipeline doing it? 

I have put back the proper templates from the support/scalable git branch.

Also, I asked for UserData to be used for the comment, not in the description of the parameter. Remove it. It needs to be right next to the  value. If thats not possible then its not required. 

You've also made MaxInstanceLifetime a parameter of the MainTemplate, yet its not used in the template. That is causing errors as it has no default value. Please remove it.